### PR TITLE
Migrating to the terraform plugin SDK.

### PR DESCRIPTION
**Pull request by lazzurs**
Migrating to the terraform plugin SDK using the automated tool. Build
and test passes and no issues with the migration tool.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,8 +3,10 @@ module github.com/terraform-providers/terraform-provider-triton
...
 	github.com/hashicorp/errwrap v1.0.0
 	github.com/hashicorp/go-multierror v1.0.0
-	github.com/hashicorp/terraform v0.12.10
+	github.com/hashicorp/terraform-plugin-sdk v1.1.0
 	github.com/joyent/triton-go v1.7.0
 	github.com/mitchellh/hashstructure v1.0.0
 	github.com/pkg/errors v0.8.1
 )
+
+go 1.13
```
Comment (twhiteman) [inline go.mod]: I don't think we need the minimum go version line here, or if we do it should be "go 1.12".

Comment (SamWhited) [reply go.mod]: Keep it, otherwise the Go tooling will add it back every time. Go 1.12 is the earliest supported version of Go, and likely what you want.

Comment (lazzurs) [reply go.mod]: Will change this to go 1.12.

**Comment by twhiteman** [comment]
Thanks for the PR.

**Pull request merged**